docs/reference-guide/triggers.md

```diff
@@ -48,7 +43,7 @@ SET object.updated_at = timestamp();
 The query may seem complex, so let's break it down:
```

Comment:
I am not a fan of this flow where we have a general format, then give an example and explain it, then explain the general stuff again

I would prefer:
1. general format
2. Explain general format and each possible case
3. Give an example and explain the example

But if you don't agree I can live with this :) But this sentence "You can find detailed explanations for each part of the trigger [further down](#trigger-name)." is bugging me... why not explain it to me right now, that's why I'm here

Reply:
I agree, revamped it a bit.